R/print.R

```diff
@@ -4,16 +4,18 @@
 #' @description
 #' Prints a brief summary of the CRE object
 #'
-#' @param x A CRE object from running the CRE function
-#' @param ... Additional arguments passed to customize the plot.
```

Comment:
Please return back `...`. `print` is a generic function with numerous parameters.